naver-searchad-api.gemspec

```diff
@@ -7,7 +7,7 @@ Gem::Specification.new do |spec|
   spec.name          = 'naver-searchad-api'
   spec.version       = Naver::Searchad::Api::VERSION
   spec.authors       = ['Min Kim']
-  spec.email         = ['developers@forward3d.com', 'min.kim@forward3d.com']
+  spec.email         = %w[developers@forward3d.com min.kim@forward3d.com]
 
   spec.summary       = %q{Naver Searchad API ruby client}
```

Comment:
https://github.com/bbatsov/ruby-style-guide#percent-literal-braces, no need to use same styles, since they are different. `%w` is an array, so makes more sense using `[]`, for %q, it is a string, so use `()`